sql/migrations/20200611182015_world.sql

```diff
@@ -12,30 +12,23 @@ INSERT INTO `migrations` VALUES ('20200611182015');
 -- npc's should roam
 UPDATE `creature` SET `movement_type` = 1, `wander_distance` = 3 WHERE `guid` IN (4248, 4181, 4064, 4245, 4180);
 
-
 -- npc's should roam
 UPDATE `creature` SET `movement_type` = 1, `wander_distance` = 0.5 WHERE `guid` IN (4242, 4198, 4196, 4197, 4081, 4175, 4195, 4205, 4083, 4210, 4188, 4189, 4190);
 
-
 -- npc's should roam
 UPDATE `creature` SET `movement_type` = 1, `wander_distance` = 1 WHERE `guid` IN (5960, 4179, 4182, 4244, 4194, 4252, 4251);
 
-
--- viktori prism antras - speak
-UPDATE `creature_template` SET `ai_name` = "EventAI" WHERE entry = 4198;
-INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (419802, 4198, 0, 1, 0, 100, 1, 1000, 15000, 150000, 180000, 419802, 0, 0, 'Viktori Prism\\\'Antras - speak');
-INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (419802, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 77, 79, 0, 0, 0, 0, 0, 0, 0, 'Viktori Prism\'Antras - speak');
-
+-- Braelyn Firehand - speak
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (419802, 4198, 0, 1, 0, 100, 1, 1000, 15000, 150000, 180000, 419802, 0, 0, 'Braelyn Firehand - Say Text OOC');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (419802, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 77, 79, 0, 0, 0, 0, 0, 0, 0, 'Braelyn Firehand - Say Text');

```

Comment:
These texts should be spoken by Viktori Prism Antras, apparently GUID 4198 was mistakenly set instead of EntryID 276 in the script ID.